### PR TITLE
New Strings - New purchase flow

### PR DESCRIPTION
**Pull request by carlosgarridoaptoide**
**What does this PR do?**

   A few sentences describing the overall goals of the pull request's commits.

   Explain any technical decisions that were made and why (when it makes sense).

**Database changed?**

   Yes | No

**Where should the reviewer start?**

- [ ] Foo.java
- [ ] Foo.kt

**How should this be manually tested?**

  Flow on how to test this.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-XXXX](https://aptoide.atlassian.net/browse/APPC-XXXX)

**Code Review Checklist**

- [ ] Architecture
- [ ] Documentation on public interfaces
- [ ] Database changed?
- [ ] Database migration (if applicable)?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] Functional QA tests pass

### CODE REVIEW

File: ui/common/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -750,6 +756,17 @@
   <string name="purchase_bank_transer_secure_body">Secure transfer directly from your bank.</string>
   <string name="purchase_bank_transfer_pending_disclaimer">We\'re waiting for the funds to arrive from your bank.</string>
 
+  <!--New Purchase flow-->
+  <string name="purchase_add_method_disclaimer">By adding a payment method you agree and consent to be supplied with the content upon payment and thereby lose your right of withdrawal.</string>
```
Comment (inesgoldschmidt) [inline ui/common/src/main/res/values/strings.xml:760]: this string is not required, we'll keep the existing one 
´´´
By proceeding with the payment, you agree and consent to be supplied with the content upon payment and thereby lose your right of withdrawal.
´´´

File: ui/common/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -569,6 +569,12 @@
   <string name="max">Max</string>
   <string name="amount">Amount</string>
   <string name="total_title">Total</string>
+  <string name="add_payment_method_button">Add payment method</string>
```
Comment (inesgoldschmidt) [inline ui/common/src/main/res/values/strings.xml:572]: @carlosgarridoaptoide please also add the string 'Change payment method' link to layout [here](https://www.figma.com/design/d7JmvYOmLDv3Fb6H9VTOPE/%F0%9F%93%A6-Wallet-Android---Layouts?node-id=719-6334&node-type=frame&t=I9UcxPlXRC1J8fip-0)

**Review by inesgoldschmidt — CHANGES_REQUESTED** [review]

File: ui/common/src/main/res/values/strings.xml (hunk 2 above)
Comment (carlosgarridoaptoide) [reply ui/common/src/main/res/values/strings.xml:572]: Done

File: ui/common/src/main/res/values/strings.xml (hunk 1 above)
Comment (carlosgarridoaptoide) [reply ui/common/src/main/res/values/strings.xml]: Done

**Review by rodolfobcardoso — APPROVED** [review]

**Pull request merged**